### ISSUE TITLE
intestinal fistula

### ISSUE THREAD

**Issue by OOPD-LJF** [monarch-initiative/mondo#5543]
**Preferred term label: intestinal fistula**

**Synonyms: gastrointestinal fistula; pancreatic fistula**

**Definition (PMID: 30480947): a fistula that connects the intestine to an adjacent organ or surface. Common causes of intestinal fistulas include surgical procedures, diverticular disease, inflammatory bowel disease, malignancy, radiation, and injury due to trauma or foreign bodies.**

**Parent term: intestinal disorder**

**Children terms N/A**

**Your nano-attribution (ORCID): 0000-0001-7151-1615**

**Comment by pnrobinson** [comment]
This seems more a feature than a disease entity.

**Comment by sabrinatoro** [comment]
"intestinal fistula" could be a phenotype. However, NCIT (NCIT:C34732) refers to this term as a "disease or disorder". We also have many other "fistula" terms (also considered "disease or disorder" in NCIT and other databases. Therefore I added this term into Mondo.

**Comment by pnrobinson** [comment]
I think this is a phenotypic feature.
See: https://hpo.jax.org/app/browse/term/HP:0100819
also -- https://emedicine.medscape.com/article/179444-differential
https://www.ncbi.nlm.nih.gov/books/NBK534208/

**Comment by sabrinatoro** [comment]
Thank you @pnrobinson . 
I will make an issue to review all the "fistula" terms we have in Mondo.
